src/Parser/Serialization/ISerialisation.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Text;
+
+namespace Parser
+{
+    public interface ISerialisation
```

Comment:
`ISerialization` would be more consistent with method name (or vice-versa, but .NET uses `z`).